forge/terrain/topology.py

```diff
@@ -53,13 +56,20 @@ def addVertices(self, vertices):
             lookupKey = ','.join([str(vertex[0]), str(vertex[1]), str(vertex[2])])
             faceIndex = self._lookupVertexIndex(lookupKey)
             if faceIndex is not None:
+                # Sometimes we can have triangles with zero area (due to unfortunate clipping)
+                # In that case skip them
+                if faceIndex in face:
+                    break
```

Comment:
Fishy no? I recreated the tiles for 8 to 14 as I think we now have border atefacts with this approach.